README.md

```diff
@@ -597,6 +597,21 @@ normal circumstances.
 
 Default: `${project.basedir}/target/habushu.placeholder.txt`
 
+
+#### distDirectory ####
```

Comment:
Q/S: It seems like this should be responsible for more than just the clean plugin functionality. Wouldn't we want to use the same property to drive _where_ Habushu generates artifacts, whether distribution or intermediary/target artifacts?